distributedcombigrid/src/sgpp/distributedcombigrid/manager/ProcessManager.cpp

```diff
@@ -396,6 +396,32 @@ std::vector<double> ProcessManager::evalErrorOnDFG(const LevelVector& leval, siz
   return g->evalErrorOnDFG(leval);
 }
 
+std::vector<CombiDataType> ProcessManager::interpolateValues(const std::vector<std::vector<real>>& interpolationCoords) {
+  auto numValues = interpolationCoords.size();
+  std::vector<std::vector<CombiDataType>> values (pgroups_.size(), std::vector<CombiDataType>(numValues));
+  std::vector<MPI_Request> requests(pgroups_.size());
+  for (size_t i = 0; i < pgroups_.size(); ++i) {
+    pgroups_[i]->interpolateValues(interpolationCoords, values[i], requests[i]);
+    // MPI_Wait(&requests[i], MPI_STATUS_IGNORE); // even this won't make it work
```

Comment:
grrr, if I call MPI_Wait DIRECTLY AFTER calling `pgroups_[i]->interpolateValues`, I get the error. But if it is the LAST thing I do INSIDE the function, it magically works...?!?